numpy/linalg/_linalg.py

```diff
@@ -2587,7 +2587,7 @@ def norm(x, ord=None, axis=None, keepdims=False):
         Input array.  If `axis` is None, `x` must be 1-D or 2-D, unless `ord`
         is None. If both `axis` and `ord` are None, the 2-norm of
         ``x.ravel`` will be returned.
-    ord : {non-zero int, inf, -inf, 'fro', 'nuc'}, optional
+    ord : {int, inf, -inf, 'fro', 'nuc'}, optional
```

Comment:
This works for vectors, but have you tested with matrices? 
```python3
import numpy as np
x = np.ones((3, 3))
np.linalg.norm(x, ord=0)
# ValueError: Invalid norm order for matrices.
```

Reply:
Then again, I suppose it's better to say any int here and clarify later, which the table does. Never mind.

Reply:
```suggestion
    ord : {int, float, inf, -inf, 'fro', 'nuc'}, optional
```